### PR TITLE
feat: add /oauth proxying in webpack dev server

### PR DESCRIPTION
**Pull request by goller**
- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass
- [x] http/swagger.yml updated (if modified Go structs or API)
- [x] Documentation updated or issue created (provide link to issue/pr)
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -17,8 +17,10 @@
   },
   "scripts": {
     "start": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run build:vendor && yarn run start:dev",
-    "start:docker": "yarn generate && yarn build:vendor && yarn run start:dev",
+    "start:cloud": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run build:vendor && yarn run start:cloud-dev",
     "start:dev": "webpack-dev-server --config ./webpack.dev.ts",
+    "start:cloud-dev": "CLOUD=true CLOUD_LOGOUT_URL=http://localhost:8080/api/v2/signout CLOUD_URL=http://localhost:9999 webpack-dev-server --config ./webpack.dev.ts",
```
Comment (hoorayimhelping) [inline ui/package.json]: thoughts on `dev-cloud` vs `cloud-dev`?

I'm thinking

`start:dev`
`start:dev-cloud`
`start:dev-robot_dragon`

might be very logical and linear

File: ui/webpack.dev.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ module.exports = merge(common, {
     historyApiFallback: true,
     compress: true,
     proxy: {
+      '/oauth': 'http://localhost:9999',
```
Comment (hoorayimhelping) [inline ui/webpack.dev.ts]: what do you think about alphabetization here, absent any other ordering scheme?

File: ui/package.json (hunk 1 above)
Comment (drdelambre) [inline ui/package.json]: `CLOUD=true` is no longer required. Should we also be more consistent with the cross-env usage? Seems weird that `start:cloud` is using it to be windows compliant, but `start:cloud-dev` isn't

**Comment by goller** [comment]
@drdelambre @hoorayimhelping I updated ! Please take a look

**Pull request merged**